custom_components/tuya_local/devices/hotshot2000_heater.yaml

```diff
@@ -12,14 +12,7 @@ primary_entity:
         - dps_val: false
           value: "off"
         - dps_val: true
-          constraint: work_mode
-          conditions:
-            - dps_val: level1
-              value: fan_only
-            - dps_val: level2
-              value: heat
-            - dps_val: level3
-              value: heat
+          value: heat
```

Comment:
I think this was OK to leave in, unless you observed problems with it.

Reply:
Ok, then level1 will be fan_only under hvac_mode and none + hidden under preset_mode. This looks logical in the UI with built-in icons for all modes and presets.